airbyte/constants.py

```diff
@@ -56,6 +56,41 @@
 your mounted Google Drive by setting this to a path like `/content/drive/MyDrive/Airbyte/cache`.
 """
 
+DEFAULT_PROJECT_DIR: Path = (
+    Path() / ".airbyte"
+    if "AIRBYTE_PROJECT_DIR" not in os.environ
+    else Path(os.environ["AIRBYTE_PROJECT_DIR"])
+)
```

Comment:
Let's handle relative directories and home directories here upfront.

Maybe add expanduser() and/or absolute()?